app/helpers/application_helper.rb

```diff
@@ -25,6 +26,17 @@ def logo_image
 
   private
 
+  def current_user_from_header
+    # current_auth_user is not defined if the including class does not
+    # include Knock::Authenticable. Hence we need to check if it is
+    # actually defined.
+    # @note We need to call this function first, and then fail. We cannot just
+    #   check if it is defined, as it is generated automatically.
+    current_auth_user&.person
+  rescue StandardError => _e
```

Comment:
What about all the errors that the JWT authentication procedure could throw? Are they some subclass of `StandardError`? Because then we are dropping a lot of good information here. Is there a way to check whether Knock::Authenticable was included? e.g., check what methods it defines and see if they're defined in the current context? Because I would prefer that over silencing all standard errors.